cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalFunctionsTest.scala

```diff
@@ -0,0 +1,1317 @@
+            |   duration({days: 14, seconds: 70, milliseconds: 1}),
+            |   duration({minutes: 1.5, seconds: 1})] AS d
+            |RETURN d""".stripMargin, ResultAssertions((r) => {
+            //CYPHER_TODO
+          })) {
+          resultTable()
+        }
+      }
+      section("Creating a _Duration_ from a string", "functions-duration-create-string") {
+        p(
+          """`duration()` returns the _Duration_ value obtained by parsing a string representation of a temporal amount.""".stripMargin)
+        function("duration(temporalAmount)", "A Duration.", ("temporalAmount", "A string representing a temporal amount."))
+        considerations("`temporalAmount` must comply with either the <<cypher-temporal-specifying-durations, unit based form or date-and-time based form defined for _Durations_>>.")
+        query(
+          """UNWIND [duration("P14DT16H12M"),
+            |   duration("P5M1.5D"),
+            |   duration("P0.75M"),
+            |   duration("PT0.75M"),
+            |   duration("P2.5W"),
+            |   duration("P12Y5M14DT16H12M70S"),
+            |   duration("P2012-02-02T14:37:21.545")] AS d
+            |RETURN d""".stripMargin, ResultAssertions((r) => {
+            //CYPHER_TODO
+          })) {
+          resultTable()
+        }
+      }
+      section("Computing the _Duration_ between two temporal instants", "functions-duration-computing") {
+        p(
+          """`duration()` has sub-functions which compute the _logical difference_ (in days, months, etc) between two temporal instant values:
+            |
+            |* `duration.between(a, b)`: Computes the difference in multiple components between instant `a` and instant `b`.
+            |* `duration.inMonths(a, b)`: Computes the difference in whole months (or quarters or years) between instant `a` and instant `b`.
+            |* `duration.inDays(a, b)`: Computes the difference in whole days (or weeks) between instant `a` and instant `b`.
+            |* `duration.inSeconds(a, b)`: Computes the difference in seconds (and fractions of seconds, or minutes or hours) between instant `a` and instant `b`.
+            |""".stripMargin)
+        section("duration.between()", "functions-duration-between") {
+          p(
+            """`duration.between()` returns the _Duration_ value equal to the difference between the two given instants.""".stripMargin)
+          function("duration.between(instant~1~, instant~2~)", "A Duration.", ("instant~1~", "An expression returning any temporal instant type (_Date_ etc) that represents the starting instant."), ("instant~2~", "An expression returning any temporal instant type (_Date_ etc) that represents the ending instant."))
+          considerations("If `instant~2~` occurs earlier than `instant~1~`, the resulting _Duration_ will be negative.")
+          query(
+            """UNWIND [duration.between(date("1984-10-11"), date("2015-06-24")),
```

Comment:
Here, again, I find these many examples in one query to be hard to comprehend for documentation users.